b2sdk/api.py

```diff
@@ -85,6 +86,11 @@ class handles several things that simplify the task of uploading
     BUCKET_CLASS = staticmethod(Bucket)
     SESSION_CLASS = staticmethod(B2Session)
 
+    @classmethod
+    def file_version_factory(cls):
```

Comment:
I am wondering - after this change FileVersion would be "bound" to the api **instance** that "initialized/created" it - so why we want it to still have "instanceless" interface - we can initialize factory with api **instance** and make it a regular **atribute** - there would be not need to pass `self.api` or whatever in every place in the code - code would look nicer I think